src/Components/TodoStatus/TodoStatus.tsx

```diff
@@ -0,0 +1,45 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { Status } from '../../types/Status';
+
+type Props = {
+  status: Status;
+  onStatusChanged: (status: Status) => void;
+};
+
+export const TodoStatus: FC<Props> = ({
+  status,
+  onStatusChanged,
+}) => (
+  <nav className="filter">
+    <a
+      href="#/"
```

Comment:
Prevent the default behavior of the anchor tag to avoid navigating away from the page.